tests/components/dynalite/test_bridge.py

```diff
@@ -51,7 +50,7 @@ def dyn_bridge():
     device2.category = "switch"
     dyn_bridge.add_devices_when_registered([device1, device2])
     reg_func = Mock()
-    dyn_bridge.register_add_devices(reg_func)
+    dyn_bridge.register_add_devices("light", reg_func)
```

Comment:
We should rewrite these tests so they are not aware of bridge details.

Here we want to test that that a light entity state exists after first adding two devices, one light and one switch, and then setting up platforms.

1. Create a mock config entry.
2. Patch the device library class, to get access to the callback for new devices.
3. Set up the integration with `hass.config_entries.async_setup` passing the mock entry id.
4. Use the new device callback and call it with the two mock devices, one light and one switch.
5. Await `hass.async_block_till_done` to let the integration set up the platforms.
6. Assert that there's a light entity state as expected.

Reply:
done